target/linux/ipq807x/base-files/etc/board.d/02_network

```diff
@@ -20,6 +45,10 @@ ipq807x_setup_interfaces()
 	edimax,cax1800)
 		ucidef_set_interfaces_lan_wan "lan"
 		;;
+	netgear,wax218)
+		ucidef_set_interface_lan "eth0" "dhcp"
```

Comment:
Outdated naming, use port labels.

Reply:
You also have to remove `dhcp` because having only one port makes it inaccessible

Reply:
@Fil010, since this device is an AP, not a router, it doesn't make sense for it to start issuing IP addresses. It's intended to be dropped into an existing network, and even the vendor FW has the behavior of requesting an IP rather than handing one out.

I've been bit by this pretty hard during development when this device came up with 192.168.1.1 and started responding to ARP requests, bringing down my entire LAN. I see other APs in OPenWRT using "dhcp", so I didn't think this would be an issue.

Reply:
Very true, what you comment more than an issue is like a thought, since there may be a person who can configure it as he wishes